ui-admin/src/portal/images/SiteImageUploadModal.tsx

```diff
@@ -11,7 +11,7 @@ import { useFileUploadButton } from 'util/uploadUtils'
 
 
 export const allowedImageTypes = ['gif', 'ico', 'jpeg', 'jpg', 'png', 'svg', 'webp']
-export const allowedDocumentTypes = ['pdf']
+export const allowedDocumentTypes = ['pdf', 'json']
```

Comment:
let's go ahead and add .txt and .csv here -- I'm sure we'll want them eventually